tests/find/test-suite-1/test.zzz-suite-1.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-['http', 'local'].forEach(function (adapter) {
+['local', 'http'].forEach(function (adapter) {
```

Comment:
Sorry this shouldn't have made it through, I'll revert.

I switched it because my confidence in local passing is lower than my confidence in http passing, so this way failures in my code came faster.